lib/xnvme_be_fbsd_nvme.c

```diff
@@ -16,7 +16,6 @@ __FBSDID("$FreeBSD$");
 #include <unistd.h>
 #include <dev/nvme/nvme.h>
 #include <errno.h>
-#include <libxnvme_spec_fs.h>
```

Comment:
This should include libxnvme.h

Reply:
Since the internal headers still include other headers, then it gets ``libxnvme.h`` via ``xnvme_be.h``